fwdpp/recombination.hpp

```diff
@@ -47,6 +47,25 @@ namespace KTfwd
             pos.emplace_back(std::numeric_limits<double>::max());
             return pos;
         }
+
+        template <typename gamete_t, typename mcont_t>
```

Comment:
The idea of a static member can probably be dropped entirely once the `operator()` no longer needs to now about gametes and whatnot.  Given that most sims in practice keep the rng, recomb. rate, and position intervals constant over time, these data should be kept as const members.